tests/bugfixes/redmine/test_issue_855.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+
+from system_tests import CaseMeta, path
+
+
+class SegfaultWhenAccessingFocalLength(metaclass=CaseMeta):
```

Comment:
[suggestion] Same comment here. I guess the application was crashing before, but now we are controlling the situation.